spec/features/open_course_creation_spec.rb

```diff
@@ -14,6 +14,8 @@ def fill_out_open_course_creator_form
 describe 'open course creation', type: :feature, js: true do
   let(:user) { create(:user) }
   before do
+    system_time_zone = Time.zone
```

Comment:
I think this needs to be either an instance variable or it needs to be outside the 'before' block so it is in scope for the 'after' block.